importers/exchanges/binance.js

```diff
@@ -22,15 +22,15 @@ var fetch = () => {
 };
 
 var handleFetch = (unk, trades) => {
-  if (trades.length > 0) {
-    var last = moment.unix(_.last(trades).date).utc();
-    var next = last.clone();
+  if (trades.length > 0 && lastTid < _.first(trades).tid) {
+    var lastTid = _.last(trades).tid;
+    var next = moment.unix(_.last(trades).date).utc();
```

Comment:
Don't use the timestamp of the last trade, if there are no trades within a 1 hour period (which does in fact happen) then it will get stuck here. Please see my PR with this fix, it addresses this issue.

Reply:
Actually I see that this bug is also still present in my PR... I made this change somewhere, and have lost it. I'll reapply to my branch momentarily

Reply:
right, it could return empty...in that case we need to go to another hour, I'll push a fix

Reply:
if there are no trades it will just go to another hour, as expected =]
Hence first check:
`trades.length > 0`